nodes/IfNode/IfNode.service.ts

```
@@ -0,0 +1,193 @@
+    switch (operator) {
+      case 'equals':
+        return actualStr === expectedStr
+
+      case 'notEquals':
+        return actualStr !== expectedStr
+
+      case 'contains':
+        return actualStr.toLowerCase().includes(expectedStr.toLowerCase())
+
+      case 'greaterThan':
+        const actualNum = Number(actualValue)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)